Makefile

```diff
@@ -269,7 +269,7 @@ _run-%:
 	npx react-native run-$(SYSTEM)
 
 # TODO: Migrate this to a Nix recipe, much the same way as nix/mobile/android/targets/release-android.nix
-run-android: export TARGET := android
+run-android: export TARGET := default
```

Comment:
could you pls elaborate on how this change works?

Reply:
It's pretty simple. There's a `nix/shells.nix` file that defines shells we use in CI and `Makefile`:
https://github.com/status-im/status-react/blob/afa2a92045ff171e7065864b64b13a693262e500/nix/shells.nix#L51-L71
These shells are defined using the `mkShell` derivation, here's a simple example:
https://github.com/status-im/status-react/blob/afa2a92045ff171e7065864b64b13a693262e500/nix/shells.nix#L25-L28
Which creates a the `keytool-sh` derivation that is then exported as `keytool` which includes `openjdk8` in `PATH` when created using `make shell TARGET=keytool`:
```
 > make shell TARGET=keytool
Configuring default Nix shell for target 'keytool'...

[nix-shell:~/work/status-react]$ which keytool
/nix/store/88zs3fkbwq5vxrynmc54j7s03vbyfbxw-openjdk-8u222-ga/bin/keytool
```
The `default` target shell is just one that includes all the tools we usually use:
https://github.com/status-im/status-react/blob/afa2a92045ff171e7065864b64b13a693262e500/nix/shells.nix#L30-L43